packages/core/src/data-grid/data-grid.tsx

```diff
@@ -1000,13 +1013,21 @@ const DataGrid: React.ForwardRefRenderFunction<DataGridRef, DataGridProps> = (p,
     const onDragStartImpl = React.useCallback(
         (event: DragEvent) => {
             const canvas = ref.current;
-            if (canvas === null || !isDraggable) return false;
+            if (canvas === null || isDraggable === false || isResizing) {
+                event.preventDefault();
+                return false;
```

Comment:
Why this `return false`? I don't think this does anything if you use `addEventListener`. Same for all the other ones.